packages/core/src/components/BaseInput/BaseInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { forwardRef } from "react";
+import cx from "classnames";
+import styles from "./BaseInput.module.scss";
+import { BaseInputComponent } from "./BaseInput.types";
+import { getStyle } from "../../helpers/typesciptCssModulesHelper";
+
+const BaseInput: BaseInputComponent = forwardRef(
+  (
+    {
+      size = "medium",
+      leftRender,
+      rightRender,
+      withReadOnlyStyle,
+      success,
+      error,
+      wrapperRole,
+      inputRole,
```

Comment:
Can they get two different roles?

Reply:
yep, for example in combobox, the wrapper should be "search" and the input "combobox" (it sounds weird, I know)